vividus/src/main/java/org/vividus/bdd/steps/BddVariableSteps.java

```diff
@@ -177,18 +180,41 @@ public boolean compareVariables(Object variable1, ComparisonRule condition, Obje
      * @param table ExamplesTable with parameters to compare with
      */
     @Then("`$variable` is equal to table ignoring extra columns:$table")
+    @SuppressWarnings("unchecked")
     public void tablesAreEqualIgnoringExtraColumns(Object variable, ExamplesTable table)
     {
-        if (!(variable instanceof Map))
-        {
-            throw new IllegalArgumentException("'variable' should be instance of map");
-        }
-        Map<String, String> expectedMap = MapUtils.convertSingleRowExamplesTableToMap(table);
-        List<EntryComparisonResult> results = ComparisonUtils.checkMapContainsSubMap((Map<?, ?>) variable, expectedMap);
+        isTrue(variable instanceof Map, "'variable' should be instance of map");
```

Comment:
out of scope but maybe The 'variable' ...

Reply:
you can consult experts 😄  but AFAIK `'variable'` is a name of the input parameter, so it doesn't require articles